eng/Versions.props

```diff
@@ -8,4 +8,21 @@
     <PreReleaseVersionLabel>alpha</PreReleaseVersionLabel>
     <PreReleaseVersionIteration>1</PreReleaseVersionIteration>
   </PropertyGroup>
+
+  <PropertyGroup>
+    <AzureActiveDirectoryIdentityModelExtensionsExternalPackageVersion>7.1.2</AzureActiveDirectoryIdentityModelExtensionsExternalPackageVersion>
```

Comment:
An alternative suggestion on the name that is a bit shorter and decouples the version from just the packages - {repoName}ReleaseVersion